WordPress/gutenberg
# Fix file block copy paste text formatting issue

## What?

Remove formatting when a text is copy-pasted into the file block's text input field, which is used for the file name.

This is a PR refresh and replaces [38198](https://github.com/WordPress/gutenberg/pull/38198/). Props @aurooba .

Closes [10967
](https://github.com/WordPress/gutenberg/issues/10967)

## Why?

When a formatted text, for example a text that includes a link, is copied and pasted into the input, there is no way to remove the formatting.

## How?

Replaces RichText with PlainText.

## Testing Instructions
1. Add a file block
2. Remove the text that is in the file name field. Confirm that the placeholder text is showing.
2. Copy this text: [File name](http://example.org/)
3. Paste the text into the file name field.
4. Confirm that it is entered as plain text without the link.

## Screenshots or screencast
